### ISSUE TITLE
Unable to restore beta on "latest" docker image on raspberry pi

### ISSUE THREAD

**Issue by andsoitgoes** [homebridge/docker-homebridge#504]
### Describe The Bug

When I try to update to homebridge:latest, I can't restore from backup. The last available image this worked on was the November '22 image

The second I run the CLI docker command to install the "latest" version, it hangs when I try to upload the backup file

### Docker Config

```yml
docker create \
 --net=host \
 --name=homebridge \
 --restart=unless-stopped \
 -v $(pwd)/xxx:/XXX\
oznu/homebridge:2022-11-14

also 

docker create \
 --net=host \
 --name=homebridge \
 --restart=unless-stopped \
 -v $(pwd)/xxx:/XXX\
oznu/homebridge:latest
```

### Logs

```text
No logs showing this error, just hangs when trying to upload the backup, the logs show nothing
```

### Host Operating System

Raspberry Pi OS - Latest

### Host Architecture

x86_64 / amd64

**Comment by Spy-1** [comment]
me to.
Same Problem!

**Comment by andsoitgoes** [comment]
Hoping someone else can help with this, fingers crossed!

**Comment by andsoitgoes** [comment]
> We have have updated the docker image and this issue with restore is now resolved.

I wish I could say that it is, but I'm having the exact same issues. It still hangs when I load the backup

**Comment by NorthernMan54** [comment]
This seems odd, as I tried that during testing and was successful.

could the backup image be corrupted?

**Comment by andsoitgoes** [comment]
> This seems odd, as I tried that during testing and was successful.
> 
> could the backup image be corrupted?

I don't think so - because the backup image works fine in the November release.

I was noticing something, though, the most recent release is April - so I'm a bit confused, is there a release I'm not seeing? I thought I might need to try the dev branch, but I can't imagine that would be necessary.

I just know that when I load up the Nov image, the backup loads immediately. The second I try that same backup with the "latest" tag, it fails.

**Comment by NorthernMan54** [comment]
@andsoitgoes In May we refreshed the docker image, and moved the image hosting to homebridge/homebridge.  That may explain why you can't see the updated image.

**Comment by andsoitgoes** [comment]
Ahh, that makes so much sense. I'll give it a go and see if that works. Thanks!
